### PR TITLE
cleanup info about external repositories/projects

### PR DESCRIPTION
**Pull request by JacekDanecki**
Remove links to inactive projects on:
- Copr
- Launchpad
- Semaphore CI
- SourceForge

Remove scripts to build on Semaphore CI and Travis

Signed-off-by: Jacek Danecki <jacek.danecki@intel.com>

### CODE REVIEW

File: level_zero/doc/DISTRIBUTIONS.md [hunk 1]
```diff
@@ -6,65 +6,9 @@ SPDX-License-Identifier: MIT
...
-
-## Ubuntu* ppa for 18.04, 20.04, 21.04
-
-```
-add-apt-repository ppa:intel-opencl/intel-opencl
-apt update
-apt install libze-loader libze-intel-gpu
-```
-
 ## Building and installation
```
[the remark this answers is not in the document]
Comment (JacekDanecki) [reply level_zero/doc/DISTRIBUTIONS.md]: I've removed this file in commit [2adcc5a](https://github.com/intel/compute-runtime/pull/511/commits/2adcc5aa784eef3295480a996c1dbdb294ca9561)

**Comment by eero-t** [comment]
This removes the outdated information, but it also does not mention any other repository from where one could get latest builds. What one should use now to get the pre-built versions of the latest releases (e.g. for Ubuntu)?

**Comment by JacekDanecki** [comment]
I'm leaving compute team, so wait for answer from the team.

**Comment by JablonskiMateusz** [comment]
change merged: https://github.com/intel/compute-runtime/commit/69ca6f9cfba34ec8aca296eb5dcce51cceb8281a

**Comment by PiotrRozenfeld** [comment]
> This removes the outdated information, but it also does not mention any other repository from where one could get latest builds. What one should use now to get the pre-built versions of the latest releases (e.g. for Ubuntu)?

For Ubuntu, releases section of this github project should be sufficient. For other distros, the default channel to get the binaries should be repositories.intel.com (see instructions on https://dgpu-docs.intel.com/). We are aware that that release channel is currently lagging behind, the intention is to improve on it in the near future.

**Pull request closed without merging**